lidar_pose:
  position_xy:
  - -0.7008517409048839
  - -3.67140434155935
  rotation_z_deg: 101.81536617022307
transformation_matrix:
- - -0.20475856718758137
  - -0.9788125097092337
  - -0.7008517409048839
- - 0.9788125097092337
  - -0.20475856718758137
  - -3.67140434155935
